guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>The ArcGIS Platform, including the ArcGIS API for Python, manages and transforms geographic data with a large suite of tools and functions collectively known as <a href="http://pro.arcgis.com/en/pro-app/help/analysis/geoprocessing/basics/what-is-geoprocessing-.htm" target="_blank">geoprocessing</a>. The GeoAnalytics Tools in the ArcGIS API for Python are a subset of geoprocessing tools that operate in the context of a <a href="https://developers.arcgis.com/python/guide/advanced-concepts/#Geoprocessing-environment" target="_blank">geoprocessing environment</a>. You can set various aspects of this environment to control how tools are executed and what messages you receive during and after the execution. See the <a href="https://developers.arcgis.com/python/guide/advanced-concepts/#Logging-and-error-handling" target="_blank">Logging and error handling</a> section in the API for Python Geoprocessing Guide's <a href="https://developers.arcgis.com/python/guide/advanced-concepts/" target="_blank">Advanced concepts</a> topic for ways to control messaging, including the <code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.env.html#verbose" target="_blank">arcgis.env.verbose</a></code> setting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>The GeoAnalytics Tools in the ArcGIS API for Python are a subset of geoprocessing tools that operate in the context of a&nbsp;<a href="https://developers.arcgis.com/python/guide/advanced-concepts/#Geoprocessing-environment" target="_blank">geoprocessing environment</a>.</p>